### PR TITLE
Allow setting custom self-signed cert for ODCS

### PR DESCRIPTION
**Pull request by tkdchen**
* CLOUDBLD-4345

Signed-off-by: Chenxiong Qi <cqi@redhat.com>

### CODE REVIEW

File: docs/admins.rst [hunk 1]
```diff
@@ -229,6 +229,13 @@ default_signing_intent
   Name of the default signing intent to be used when one is not provided
   in ``container.yaml``.
 
+To allow accessing repositories generated by ODCS via HTTPS, set
```
Comment (MartinBasti) [inline docs/admins.rst:232]: That's not true, you need this to allow custom selfsigned CA certificates. It depends what kind of cert is ODCS using

File: docs/admins.rst [hunk 2]
```diff
@@ -229,6 +229,15 @@ default_signing_intent
   Name of the default signing intent to be used when one is not provided
   in ``container.yaml``.
 
+If HTTPS is enabled for the access to YUM repositories hosted in ODCS,
```
Comment (MartinBasti) [inline docs/admins.rst:232]: Not just ODCS, we have implemented it for any yum repo, even those passed by users via parameter.

Comment (tkdchen) [reply docs/admins.rst:232]: Well, if so, this is not the right section to add the new config description. This section is dedicated for odcs. Which section is proper to have a general description for the `builder_ca_bundle`?

Comment (MartinBasti) [reply docs/admins.rst:232]: Just make it as a new section see `Limiting image size` for example

Comment (MartinBasti) [reply docs/admins.rst:232]: It can be section `Custom CA bundle for yum repos` or so

**Comment by tkdchen** [comment]
@MartinBasti PTAL.

**Review by MartinBasti — APPROVED** [review]
Thank you!

**Review by mkosiarc — APPROVED** [review]

**Review by lkolacek — APPROVED** [review]

**Review by rcerven — APPROVED** [review]

**Pull request merged**